### DISCUSSION TITLE
US30 code

### DISCUSSION THREAD

**Discussion by BorderCollieLover** [bug-or-feature/bc-utils#17] [General]
In config.py there is this line: "US30": {"code": "UB", "cycle": "HMUZ", "exchange": "CME"},

Should it be UD instead of UB if it's for  US ultra bond? 
![image](https://github.com/user-attachments/assets/15d8fece-154c-4e47-a15f-acb1ffa7550a)

Thanks for sharing this repository. I definitely benefitted from your work 

Best

**Comment by bug-or-feature** [comment]
I think you're right. UB is Euro/Krone on Barchart. I think confusion arose because Interactive Brokers use UB. Submit a PR if you like, or I'll fix in next release

**Comment by bug-or-feature** [comment]
Fixed by #21
